sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,71 @@
+<div class="gc-pft row" data-wb-jsonmanager='{
+	"name": "feedback",
+	"extractor": [
+		{ "selector": "title", "path": "pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "language" },
+		{ "interface": "locationHref", "path": "submissionPage" },
+		{ "selector": "#wb-lng ul li:first-child a[lang]", "attr": "href", "path": "oppositelang" },
+		{ "selector": "meta[name=\"dcterms.type\"]", "attr": "content", "path": "themeopt" },
```

Comment:
Type is not about the theme/topic. But more about defining the type of the document, like here it will be "HTML", WebPage,....

We might look to use instead something like: foaf:topic => http://xmlns.com/foaf/0.1/#term_topic